### PR TITLE
Minor foldp docs changes

### PR DESCRIPTION
**Pull request by bbarker**

### CODE REVIEW

**Comment by OlivierBlanvillain** [comment]
:+1:, could you also update the scala doc?

**Comment by bbarker** [comment]
:+1: Done

File: monadic-rx/shared/src/main/scala/mhtml/rx.scala [hunk 1]
```diff
@@ -82,7 +82,8 @@ sealed trait Rx[+A] { self =>
   /**
    * Produces a `Rx` containing cumulative results of applying a binary
    * operator to each element of this `Rx`, starting from a `seed` and the
-   * current value, and moving forward in time.
+   * current value of the upstream `Rx`, and moving forward in time; no internal
+   * state is maintained.
    *
    * ```
    * val numbers: Rx[Int]
```
Comment (OlivierBlanvillain) [inline monadic-rx/shared/src/main/scala/mhtml/rx.scala]: The line below this one still has the `fold` instead of `foldp` typo :)

Comment (bbarker) [reply monadic-rx/shared/src/main/scala/mhtml/rx.scala]: oops, fixed

**Comment by OlivierBlanvillain** [comment]
LGTM, thanks!

**Pull request merged**